uefi/flashmastersection.go

```diff
@@ -41,9 +42,9 @@ func (m FlashMasterSection) Summary() string {
 		"    GbeRead=%v\n"+
 		"    GbeWrite=%v\n"+
 		"}",
-		m.BiosID, m.BiosRead, m.BiosWrite,
-		m.MeID, m.MeRead, m.MeWrite,
-		m.GbeID, m.GbeRead, m.GbeWrite,
+		m.BIOS.ID, m.BIOS.Read, m.BIOS.Write,
```

Comment:
same here - RegionPermissions could have a stringer?

Reply:
Actually, in general - can we skip the multi-line summaries? We should have compact fmt.Stringers, and if you want multi-line summaries like this, you should use a deep pretty-printing library like https://godoc.org/github.com/kr/pretty or https://github.com/davecgh/go-spew - reflection does a much better job than people here

Anyway, this is all optional, your judgment

Reply:
I agree, but I don't wanna remove the original summary stuff for this pr. we can do that in another one.